.github/workflows/build.yml

```diff
@@ -17,6 +17,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
+          # A complete fetch needed for spotless to ratchet from origin/develop
+          fetch-depth: 0
```

Comment:
@anitab-org/mentorship-maintainers this needs to be discussed in the open session.

Reply:
gosh, i didn't get you wanted to talk about this in the call. and now its over 🤦🏾‍♀️ 
we can discuss on zulip

Reply:
@isabelcosta this comment https://github.com/anitab-org/mentorship-android/pull/1123#issuecomment-817256746. But I think we can merge this and optimize the actions later.